CHANGE_MANAGEMENT.md

```diff
@@ -0,0 +1,98 @@
+* Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+* Informing developers of strengths and weaknesses from a new stakeholder perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Providing financial support (the project is open source, but capacity is a constant struggle)
+* Stakeholders who continue to engage with the project and its community will often become more and more involved. Such stakeholders may find themselves becoming contributors, as described in the next section.
+
+### Contributors
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as stakeholders, contributors may also find themselves doing one or more of the following:
+* Supporting new users (existing users are often the best people to support new users)
+* Reporting bugs
+* Identifying requirements
+* Providing graphics and web design
+* Programming
+* Assisting with project infrastructure
+* Writing documentation
+* Fixing bugs
+* Adding features
+
+Contributors engage with the project through the issue tracker and mailing list, or by writing or editing documentation. They submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). The mailing list is the most appropriate place to ask for help when making that first contribution.
+
+As contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for committership.
+
+### Committers
+Committers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. New committers can be nominated by any existing committer. Once they have been nominated, there will be a vote by the project management committee (PMC; see below). Committer voting is one of the few activities that takes place on the project’s private management list. This is to allow PMC members to freely express their opinions about a nominee without causing embarrassment. Once the vote has been held, the aggregated voting results are published on the public mailing list. The nominee is entitled to request an explanation of any ‘no’ votes against them, regardless of the outcome of the vote. This explanation will be provided by the PMC Chair (see below) and will be anonymous and constructive in nature.
```

Comment:
Again, we don't really have a _private management (email) list_ or a _public mailing list_.